README.md

```diff
@@ -25,6 +25,7 @@ relates to weekly certification
 - API_USER_KEY: The user key for the API
 - CERTIFICATE_DIR: The path to the client certificate (certificate must be in PFX/P12 format)
 - P12_FILE: The name of the client certificate file
```

Comment:
shall we just rename this PFX_FILE since that's more accurate and consistent? seems confusing otherwise given "PFX_" below